extmod/modlwip.c

```diff
@@ -305,10 +317,24 @@ typedef struct _lwip_socket_obj_t {
                 struct tcp_pcb **array; // if alloc != 0
             } tcp;
         } connection;
+
+        // Connected TCP sockets have a single incoming pbuf that is
+        // appended to when new data comes in.
+        struct {
+            struct pbuf *pbuf;
+        } tcp;
+
+        // UDP and raw sockets have a queue of incoming pbuf's.
+        struct {
+            uint8_t iget;
```

Comment:
```suggestion
            uint8_t iget;   // Ringbuffer read index
```

(and same for `iput`)

Reply:
Updated for both lines.